Sources/XiEditor/PluginCommand.swift

```diff
@@ -16,21 +16,9 @@ import Foundation
 
 /// A custom plugin command.
 struct Command {
-    enum RpcType {
-        case Request
-        case Notification
-
-        static func fromString(string: String) -> RpcType? {
-            switch string {
-            case "notification":
-                return .Notification
-            case "request":
-                return .Request
-            default:
-                print("malformed rpc type field \(string)")
-                return nil
-            }
-        }
+    enum RpcType: String {
+        case request = "request"
+        case notification = "notification"
```

Comment:
nit: if the case of the enum matches the string we can just remove the `"string"` value 🎉